test/junit/scala/collection/IndexedSeqTest.scala

```diff
@@ -39,6 +39,12 @@ abstract class IndexedTest[T, E] {
     }
   }
 
+  /** check that lengthCompare compares values correctly */
+  @Test def checkLengthCompare(): Unit = {
+    val test = underTest(size)
+    assert(lengthCompare(test, Integer.MIN_VALUE) > 0)
```

Comment:
Was this test previously failing?

Reply:
Yes.

`size - Int.MinValue` (also, I accidentally used Java's, oops; I'll fix that) is the same as `size + Int.MaxValue + 1`, which overflows to `size + Int.MinValue` or `Int.MinValue + size`. Since `size <= Int.MaxValue`, `Int.MinValue + size <= Int.MinValue + Int.MaxValue == -1`. However, `size > Int.MinValue`, so you ought to get a positive value.

Reply:
(I actually wrote and ran the test before fixing `ArrayOps`, just to be sure)